source/computer/fedora-setup.rst

```diff
@@ -142,6 +142,38 @@ Fortran 编译器，其提供了 ``gfortran`` 命令::
 
     $ sudo dnf install gcc-gfortran
 
+Intel 软件开发工具包
+^^^^^^^^^^^^^^^^^^^^
+
+`Intel oneAPI Toolkits <https://software.intel.com/content/www/us/en/develop/tools/oneapi.html>`__
+是英特尔最新的软件开发工具包。它也提供了 C/C++ 编译器和 Fortran 编译器（``icc`` 和 ``ifort`` 命令）。
+此外还有 MKL 数学库、MPI 并行库等。该工具包是免费的，不需要许可证。
+
+日常科研安装 Base Toolkit 和 HPC Toolkit 即可。在 Fedora 系统下，官方手册提供了多种\
+`安装方式 <https://software.intel.com/content/www/us/en/develop/documentation/installation-guide-for-intel-oneapi-toolkits-linux/top.html>`__\ ，
+如在线安装、本地安装、使用 ``dnf`` 安装、使用 ``conda`` 安装等。这里，我们选择使用 ``dnf`` 安装。
+
+生成 :file:`.repo` 文件，命名为 :file:`oneAPI.repo`，并放到 :file:`/etc/yum.repos.d` 目录下::
+
+    $ cat /etc/yum.repos.d/oneAPI.repo
+    [oneAPI]
+    name=Intel(R) oneAPI repository
+    baseurl=https://yum.repos.intel.com/oneapi
+    enabled=1
+    gpgcheck=1
+    repo_gpgcheck=1
+    gpgkey=https://yum.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+
+安装工具包，默认安装目录是 :file:`/opt/intel/oneapi`::
+
+    $ sudo dnf install intel-basekit
+    $ sudo dnf install intel-hpckit
+
+安装完成后，按照\ `官方手册 <https://software.intel.com/content/www/us/en/develop/documentation/get-started-with-intel-oneapi-base-linux/>`__\
+配置好环境变量，就可以使用该工具包了::
+
+    $ echo "source /opt/intel/oneapi/setvars.sh >/dev/null 2>&1" >> ~/.bashrc
+
```

Comment:
It's a little confusing. Do I need to follow the "official tutorial" first and then change the `~/.bashrc`? Or I only need to update `~/.bashrc`?

Reply:
We only need to update `~/.bashrc` to use `icc` and `ifort`, but the official tutorial has more options.

Reply:
Clarify it in 570799a.